spec/index.html

```diff
@@ -351,6 +352,64 @@ <h3>Blank nodes</h3>
     </figure>
   </section>
 
+  <section id="section-triple-terms">
+    <h3>Triple terms</h3>
+
+    <p>As the basic unit of description in RDF, the triple is a simple
```

Comment:
Trying to use less words to say the same thing (likely a matter of writing style, but still, I think simplicity is important for a primer)

_As the atomic unit of description in RDF, the triple (or statement) states a directed relationship between two resources. Sometimes, we need to describe this relationship itself in more detail._

Reply:
+1 to @william-vw 's proposal

Reply:
Yes, this is better. While I also prefer "atomic" (as in "atomic proposition"), there have been some reactions to using that (since triples have components). That's why I used "basic". But I'll try this out. Keep commenting if something even better comes to mind.